contrib/librespot.user.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=An open source Spotify client
+Documentation=https://github.com/plietar/librespot
+
+[Service]
+Restart=always
+RestartSec=12
```

Comment:
`librespot.service` use 10 here.

I don't really care about the value, but it should be consistent